community/best_practices_managing_issues.md

```diff
@@ -0,0 +1,58 @@
+# Best practices for managing onnx issues
+
+## Enforce templates
+Currently there are no templates for creating issues in onnx. This makes it very difficult for community members to triage issues. Going forward templates shall be added and maintained for ONNX issues.
+
+Following templates shall be added
```

Comment:
Should this be in present tense and just list the link so the templates?

Reply:
We don't have templates yet... Once the templates are created I will add a link here.